sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -197,7 +197,7 @@ object ScalaNativePluginInternal {
       workdir
     },
     nativeLogger := streams.value.log,
-    nativeGC := "boehm",
+    nativeGC := "immix",
```

Comment:
I've just opened a PR for matrix builds over a GC implementation: #730. This will make sure that Immix is tested without being a default implementation.